src/index.html

```diff
@@ -2,13 +2,87 @@
 <html lang="en">
   <head>
     <meta charset="UTF-8">
-    <meta name="viewport"
-          content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
-    <meta http-equiv="X-UA-Compatible" content="ie=edge">
+    <meta
+      name="viewport"
+      content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0"
+    >
+    <meta
+      http-equiv="X-UA-Compatible"
+      content="ie=edge"
+    >
     <title>Moyo header</title>
-    <link rel="stylesheet" href="./style.css">
+    <link
+      rel="stylesheet"
+      href="./style.css"
+    >
   </head>
   <body>
-    <h1>Moyo header</h1>
+    <header class="header">
+      <div class="logo">
+        <a
+          href="#home"
+          class="logo"
+        >
+          <img
+            src="/images/logo.png"
+            alt="logo"
+            class="logo__img"
+          >
+        </a>
+      </div>
+      <nav class="nav">
+        <ul class="nav__list">
+          <li class="nav__item">
+            <a
+              href="#apple"
+              class="nav__link is-active"
+            >Apple</a>
+          </li>
+          <li class="nav__item">
+            <a
+              href="#samsung"
+              class="nav__link"
+            >Samsung</a>
```

Comment:
In case of multiline tag It will be more clear if you place content on separate line also
```suggestion
            >
              Samsung
            </a>
```